pkg/resourceoverrides/dns_record_set.go

```diff
@@ -26,9 +26,11 @@ import (
 )
 
 var (
-	rrdatasFieldName       = "rrdatas"
-	rrdatasRefsFieldName   = "rrdatasRefs"
-	routingPolicyFieldName = "routingPolicy"
```

Comment:
As I suggested in dns.yaml, `dnsAuthorizationsRef` shouldn't be a separate field.